### DISCUSSION TITLE
已经续费了两年专业版，但是界面显示的还是快到期了

### DISCUSSION THREAD

**Discussion by chenhong231** [getlantern/lantern-forum-cn#897] [General]
麻烦帮解决一下，我的账号是: chenhong231@outlook.com

如图是我的付款记录。

![Uploading 微信图片_20240926012840.jpg…]()

**Comment by chenhong231** [comment]
@Mike777Khoo
![微信图片_20240926012840](https://github.com/user-attachments/assets/970385b1-bb0d-4b1c-afee-d0600b6f5e1a)

**Comment by MaxMKMax** [comment]
@chenhong231 好的. 我帮您提交去处理一下.
